instance/tests/models/test_openedx_instance.py

```diff
@@ -536,103 +536,32 @@ def test_delete_instance(self, mocks, delete_by_ref, *mock_methods):
             appserver.refresh_from_db()
 
     @staticmethod
-    def _set_appserver_terminated(appserver):
-        """
-        Transition `appserver` to AppServerStatus.Terminated.
-        """
-        appserver._status_to_waiting_for_server()
-        appserver._status_to_configuring_server()
-        appserver._status_to_running()
-        appserver._status_to_terminated()
-
-    @staticmethod
-    def _set_appserver_running(appserver):
-        """
-        Transition `appserver` to AppServerStatus.Running.
-        """
-        appserver._status_to_waiting_for_server()
-        appserver._status_to_configuring_server()
-        appserver._status_to_running()
-
-    @staticmethod
-    def _set_appserver_configuration_failed(appserver):
-        """
-        Transition `appserver` to AppServerStatus.ConfigurationFailed.
-        """
-        appserver._status_to_waiting_for_server()
-        appserver._status_to_configuring_server()
-        appserver._status_to_configuration_failed()
-
-    @staticmethod
-    def _set_appserver_errored(appserver):
-        """
-        Transition `appserver` to AppServerStatus.Error.
-        """
-        appserver._status_to_waiting_for_server()
-        appserver._status_to_error()
-
-    @staticmethod
-    def _set_server_ready(server):
-        """
-        Transition `server` to Status.Ready.
-        """
-        server._status_to_building()
-        server._status_to_booting()
-        server._status_to_ready()
-
-    @staticmethod
-    def _set_server_terminated(server):
-        """
-        Transition `server` to Status.Terminated.
-
-        Note that servers are allowed to transition to ServerStatus.Terminated from any state,
-        so it is not necessary to transition to another status first.
-        """
-        server._status_to_terminated()
-
-    def _create_appserver(self, instance, status):
-        """
-        Return appserver for `instance` that has `status`
-
-        Note that this method does not set the status of the VM (OpenStackServer)
```

Comment:
This comment should be kept. Please include it in the documentation of make_test_appserver (document the new `status` parameter).